### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by osanseviero**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,30 @@
...
-More info coming soon.
+# What is this experiment about?
+This experiment is a project between Yandex and HF in which we aim to train an ALBERT model for Bengali collaboratively. That’s where you come in! Each participant can contribute by sharing their compute power, so the more we are, the faster the training is. This is achieved using the [Hivemind library](https://github.com/learning-at-home/hivemind/). You can read more about it in [this paper](https://arxiv.org/abs/2103.03239).
+
+
+# How to participate?
+
+1. As a reminder, you need to provide your Hugging Face username to be able to participate. 
+    * Current participants are already in the allowlist.
+    * New participants need to join the #albert-allowlist channel and add their username. Someone from the team will add you to the allowlist. If you see a :hourglass:  reaction, we’re on it! If you see a :white_check_mark: , you should be added by then. Feel free to reach out to us if you don’t have access.
```
Comment (mryab) [inline README.md]:
```suggestion
    * New participants need to join the #albert-allowlist channel and add their username. Someone from the team will add you to the allowlist. If you see a :hourglass:  reaction, we’re on it! If you see a :white_check_mark:, you should be added by then. Feel free to reach out to us if you don’t have access.
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,30 @@
 
 This code trains ALBERT-large-v2 model on Bengali Wikipedia and OSCAR.
```
Comment (mryab) [inline README.md]: I think we're free to remove this

File: README.md [hunk 3]
```diff
@@ -1,4 +1,30 @@
...
+    * Current participants are already in the allowlist.
+    * New participants need to join the #albert-allowlist channel and add their username. Someone from the team will add you to the allowlist. If you see a :hourglass:  reaction, we’re on it! If you see a :white_check_mark: , you should be added by then. Feel free to reach out to us if you don’t have access.
+
+2. You can join the training in few ways:
+
+* **[Kaggle](https://www.kaggle.com/yhn112/collaborative-training-d87a28)**. This option provides you a P100 and lasts longer than Colab. This requires a Kaggle account. **You must enable Internet access and switch kernel to GPU mode explicitly.**
+    * If it is stuck at "installing dependencies" for over 5 minutes, it means you changed session type too late. Simply restart with GPU / internet enabled and it should work just fine.
+    * If you ran Kaggle during last week's training, please create a new copy to update the code. On the 3 dots on the screenshot and than create a new copy (don’t click on Edit My Copy).
+
+* **[Colab](https://colab.research.google.com/github/mryab/collaborative-training/blob/main/colab_starter.ipynb)**.
```
Comment (mryab) [inline README.md]:
```suggestion
* **[Google Colab](https://colab.research.google.com/github/mryab/collaborative-training/blob/main/colab_starter.ipynb)**.
```

**Review by mryab — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (osanseviero) [reply README.md]: Done

**Pull request merged**